pkg/commands/edit/add/configmap.go

```diff
@@ -31,18 +29,18 @@ import (
 func newCmdAddConfigMap(fSys fs.FileSystem, kf ifc.KunstructuredFactory) *cobra.Command {
 	var flags flagsAndArgs
 	cmd := &cobra.Command{
-		Use:   "configmap NAME [--from-file=[key=]source] [--from-literal=key1=value1]",
+		Use:   "configmap NAME [--plugin-type=type] [--plugin-name=name] [--args=arg] [--expand=true]",
```

Comment:
What does --expand=true do?

When would expand be false?  Can't that be achieved with escapes?